File: app/src/main/java/getyourcasts/jd/com/getyourcasts/repository/remote/data/ItuneResponse.java
package getyourcasts.jd.com.getyourcasts.repository.remote.data;

import java.util.List;

/**
 * Created by chuondao on 8/9/17.
 */

public final class ItuneResponse {
    private List<Podcast> results;

    public List<Podcast> getResults() {
        return results;
    }

    public void setResults(List<Podcast> results) {
        this.results = results;
    }

    public ItuneResponse(List<Podcast> results) {
        this.results = results;
    }
}

File: app/src/main/java/getyourcasts/jd/com/getyourcasts/repository/remote/data/ItuneResponse.kt
package getyourcasts.jd.com.getyourcasts.repository.remote.data

/**
 * Created by chuondao on 7/22/17.
 */

data class EpisodeResponse (val podcastId: String,
                    val name: String,
                    val id: String)

data class Podcast(val collectionId: String,
                   val collectionName: String,
                   val artistName : String,
                   val artworkUrl100: String,
                   val releaseDate : String,
                   val trackCount: Long)

data class ItuneResponse (val results: List<Podcast>)